### ISSUE TITLE
how install "libasound-dev portaudio19-dev libportaudio2 libportaudiocpp0 libav-tools"for Cenots 7

### ISSUE THREAD

**Issue by Amforever** [Rayhane-mamah/Tacotron-2#310]
When i use "yum install -y libasound-dev portaudio19-dev libportaudio2 libportaudiocpp0 ffmpeg libav-tools"  on the Centos7 , I got some errors:
`http://people.centos.org/hughesjr/chromium/6/x86_64/repodata/repomd.xml: [Errno 14] PYCURL ERROR 22 - "The requested URL returned error: 404 Not Found"
Trying other mirror.
No package libasound-dev available.
No package portaudio19-dev available.
No package libportaudio2 available.
No package libportaudiocpp0 available.
Package ffmpeg-0.6.5-1.el6.rf.x86_64 already installed and latest version
No package libav-tools available.
`
What should I do for this?

**Comment by ZhaoZeqing** [comment]
anaconda may help you

conda install pyaudio

**Comment by Amforever** [comment]
Yes , i am using anaconda3. Just only installing pyaudio by conda, how about other dependencies ?

**Comment by begeekmyfriend** [comment]
You need [PortAudio](http://www.portaudio.com/download.html) library and `ldconfig` command.

**Comment by ZhaoZeqing** [comment]
when using "conda install pyaudio", it installed portaudio together on my server.
